pyproject.toml

```diff
@@ -52,7 +52,7 @@ lint = [
 ]
 types = [
     "mypy[reports]==1.11.2",
-    "pyright==1.1.381",
+    "pyright==1.1.382.post1",
```

Comment:
I'm not sure about it, GH release has only `1.1.382` release.
`post1` and `post2` seems like a interim releases.
Maybe they've started building every commit, but they're using wrong nomenclature

Reply:
The pip package is separate from Microsoft's releases, and this is how they're handling packaging differences. https://github.com/RobertCraigie/pyright-python/releases